controllers/ReceptionsController.js

```diff
@@ -26,3 +27,32 @@ exports.getVisitsByUserId = async (req, res) => {
     return res.status(422).send({ answer: error.message });
   }
 };
+
+exports.addNewReception = async (req, res) => {
+  const { body } = req;
+  const id = req.user;
+  const {
+    patientName, date, complaint, doctorId,
+  } = body;
+
+  if (!body) {
+    return res.status(401).send({ answer: 'No input.' });
+  }
+  if (!patientName.trim() || !date || !valiDate(date) || !complaint.trim() || !doctorId || typeof doctorId !== 'number' || doctorId < 0) {
```

Comment:
why you doubleCheck this data ? You don't need body check if you check this data

Reply:
also you need to validate if name and compliants are valid